src/js/Controllers/BCController.js

```diff
@@ -106,12 +106,7 @@ class BCController {
             case "OnAppCapabilityUpdated":
                 if (rpc.params.appCapability.appCapabilityType === 'VIDEO_STREAMING') {
                     var vsc = rpc.params.appCapability.videoStreamingCapability;
-                    var caps = [ Object.assign({}, vsc) ];
-                    delete caps[0].additionalVideoStreamingCapabilities;
-                    if (vsc.additionalVideoStreamingCapabilities) {
-                        caps = caps.concat(vsc.additionalVideoStreamingCapabilities);
-                    }
-                    store.dispatch(setVideoStreamingCapability(rpc.params.appID, caps));
+                    store.dispatch(setVideoStreamingCapability(rpc.params.appID, vsc.additionalVideoStreamingCapabilities));
```

Comment:
Check if additionalVideoStreamingCapabilities exists and has a length > 0, if not pass empty array?

Reply:
passing an empty array in https://github.com/smartdevicelink/generic_hmi/pull/345/commits/37bf9edd9720d6105971200fa1fabfb3164d97b8

I also made it so the resolution drop down only shows if there are valid options